bears/documentation/DocumentationStyleBear.py

```diff
@@ -82,9 +83,13 @@ def process_documentation(self,
                 if stripped_desc[0] != '':
                     stripped_desc.insert(0, '')
 
-            # Indent with 4 spaces.
-            stripped_desc = ('' if line == '' else ' ' * indent_size
-                             + line for line in stripped_desc)
+            # Indent with 4 spaces or a tab.
+            if use_spaces:
+                stripped_desc = ('' if line == '' else ' ' * indent_size
+                                 + line for line in stripped_desc)
+            else:
+                stripped_desc = ('' if line == '' else '    '
+                                 + line for line in stripped_desc)
```

Comment:
`indent_size` is there for a reason. so do not remove it in the else part. 

code should be something like. As we will be using tabs if it's set to false.
```
else:
    # Let it's default be 
    indent_size = 1
    stripped_desc = ('' if line == '' else '\t' * indent_size
                               + line for line in stripped_desc)
```
Also update the `indent_size` definition in the docstring that it will be set to 1 if `use_spaces` is set to false.

Reply:
Also set default. `use_spaces: True`

Reply:
okk got it